client/src/app/user-boards/[boardId]/layout.tsx

```diff
@@ -1,10 +1,20 @@
 import React, { FC } from "react";
 import BoardSidebar from "@/components/board-details/BoardSidebar";
+import { Metadata } from "next";
 
 interface LayoutProps {
   children: React.ReactNode;
   params: { boardId: string };
 }
+
+export async function generateMetadata({ params }: LayoutProps): Promise<Metadata> {
+  const { boardId } = params;
+  //const boardData = await getBoardData(boardId); todo: [JIRA OK-219] retrieve board name for title
+  return {
+    title: `User board - ${boardId} | Obelisk`,
+    description: "User board main information",
```

Comment:
Same thing here. I don't see a point in telling our user that he is looking at a "user board". Well, there is no other type of boards XD